translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+# Brazilian Portuguese translation.
+# Copyright (C) 2020
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR ctadeu@gmail.com, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: EMAIL\n"
+"POT-Creation-Date: 2017-03-14 21:32-0700\n"
+"PO-Revision-Date: 2020-10-22 17:08+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: Carlos Panato <ctadeu@gmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: pt_BR\n"
+"X-Poedit-KeywordsList: \n"
+
+#: pkg/kubectl/cmd/create_clusterrolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a ClusterRoleBinding for user1, user2, and group1 using the "
+"cluster-admin ClusterRole\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin --clusterrole=cluster-"
+"admin --user=user1 --user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t  # Criar o ClusterRoleBinding para user1, user2, e group1 utilizando o "
+"cluster-admin ClusterRole\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin —clusterrole=cluster-"
+"admin —user=user1 —user=user2 —group=group1"
+
+#: pkg/kubectl/cmd/create_rolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a RoleBinding for user1, user2, and group1 using the admin "
+"ClusterRole\n"
+"\t\t  kubectl create rolebinding admin --clusterrole=admin --user=user1 --"
+"user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t  # Criar uma RoleBinding para user1, user2, e group1 utilizando o admin "
+"ClusterRole\n"
+"\t\t  kubectl create rolebinding admin —clusterrole=admin —user=user1 —"
+"user=user2 —group=group1"
+
+#: pkg/kubectl/cmd/create_configmap.go:44
+msgid ""
+"\n"
+"\t\t  # Create a new configmap named my-config based on folder bar\n"
+"\t\t  kubectl create configmap my-config --from-file=path/to/bar\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with specified keys instead "
+"of file basenames on disk\n"
+"\t\t  kubectl create configmap my-config --from-file=key1=/path/to/bar/file1."
+"txt --from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with key1=config1 and "
+"key2=config2\n"
+"\t\t  kubectl create configmap my-config --from-literal=key1=config1 --from-"
+"literal=key2=config2"
+msgstr ""
+"\n"
+"\t\t  # Criar um novo configmap com o nome de my-config baseado na pasta "
+"bar\n"
+"\t\t  kubectl create configmap my-config —from-file=path/to/bar\n"
+"\n"
+"\t\t  # Cria um novo configmap com o nome de my-config especificando as "
+"chaves ao invés de utilizar o nome do arquivo em disco\n"
```

Comment:
Deleted my previous comment, and commenting again with something that makes more sense.

The explanation in English is also pretty confuse here, but effectively what the command does is to create a ConfigMap where each value of the keys is the content of a specific file. So maybe:

"Cria um novo configmap com o nome my-config, onde cada chave possui o valor especificado em um arquivo distinto no disco". WDYT?